lib/pushy_client/protocol_handler.rb

```diff
@@ -468,6 +460,7 @@ def send_signed_json_command(method, json)
         @command_socket_outgoing_seq += 1
         json[:sequence] = @command_socket_outgoing_seq
         message = JSON.generate(json)
```

Comment:
I might pull this generate call and test into the validate function below. That pulls all the json generation and size testing into one place.